src/index.jsx

```diff
@@ -73,13 +74,17 @@ apiClient.ensurePublicOrAuthenticationAndCookies(
   (accessToken) => {
     const { store, history } = configure();
 
-    ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
-
     if (accessToken) {
+      ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
       identifyAuthenticatedUser(accessToken.userId);
+      sendPageEvent();
     } else {
+      // This should never happen, but it does sometimes.
+      // Add logging to learn more.
+      ReactDOM.render(<ErrorPage />, document.getElementById('root'));
```

Comment:
Is there a way to just use ErrorBoundary?

Reply:
We're handling this error outside the scope of React, so not directly no.